service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/ObservationMapper.java

```diff
@@ -195,6 +218,25 @@ private CommentType prepareCommentType(Observation observation) {
         return CommentType.USER_COMMENT;
     }
 
+    private String prepareNarrativeStatement(Observation observation, String comment, String commentType, IdMapper idMapper) {
+        var narrativeStatementTemplateParameters = NarrativeStatementTemplateParameters.builder()
+            .narrativeStatementId(randomIdGeneratorService.createNewId())
+            .commentType(commentType)
+            .issuedDate(DateFormatUtil.toHl7Format(observation.getIssued().toInstant()))
```

Comment:
Why issued date instead of effective time?

Reply:
should we follow the same logic as for effective time ?
like in StatementTimeMappingUtils.prepareEffectiveTimeForObservation?

Reply:
It's slightly different for Narrative Statements. The issuedDate usually would go in HL7/XML format, like this:
```
<center value="20201215151704"/>
```

But it's just in plain text in the Narrative Statement (not in HL7/XML format), like this: `20201215151704`

Reply:
Yes, I mean following the logic, not taking the result. effectiveDateTime, of effectiveTimePeriod.start, end

Reply:
good you pointed that! it should be in human readable format in text. So it would be good to use here DateFormatUtil.toTextFormat method

Reply:
Not according to the mapping examples and the spreadsheet. It looks like this (for example):
```
<NarrativeStatement classCode="OBS" moodCode="EVN">
                                <id root="6035E061-FD34-444B-8171-DBDD243FDE1D"/>
                                <text mediaType="text/x-h7uk-pmip">CommentType:USER COMMENT
CommentDate:20201215151713

(q) - Normal - No Action</text>
                                <statusCode code="COMPLETE"/>
                                <availabilityTime value="20201215151713"/>
                            </NarrativeStatement>
```

So it's ok to leave it as it is

Reply:
ok, my mistake. sorry